src/app.ts

```diff
@@ -427,13 +429,20 @@ export function newApp({
           authMiddlewares.bearerSession,
           authMiddlewares.local
         );
-        // Create the messages service.
+
+        const thirdPartyClientFactory = getThirdPartyServiceClientFactory(
```

Comment:
This factory should use pnFetch if service is PN

Reply:
Sure